src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -193,7 +194,18 @@ public Object instanceMain(final String[] argv) {
     protected boolean parseArgs(final String[] argv) {
 
         commandLineParser = new CommandLineParser(this);
-        final boolean ret = commandLineParser.parseArguments(System.err, argv);
+        final boolean ret;
+        try{
+            ret = commandLineParser.parseArguments(System.err, argv);
+        } catch (final UserException.CommandLineException e){
+            //The CommandLineException is treated specially - we display help and no blow up
```

Comment:
I think we should print the help first and then the error message.  Otherwise finding the actually error requires screens worth of scrollback.  I'm not a fan of printing help on every command line error to begin with since I usually just made a typo and don't want a huge dump of information to clobber my screen, but if that's the behavior we've decided on I think should at least make the error the most prominent part.